x/leveragelp/keeper/msg_server_open.go

```diff
@@ -18,6 +21,21 @@ func (k Keeper) Open(ctx sdk.Context, msg *types.MsgOpen) (*types.MsgOpenRespons
 	if err := k.CheckUserAuthorization(ctx, msg); err != nil {
 		return nil, err
 	}
+	params := k.stableKeeper.GetParams(ctx)
+	moduleAddr := authtypes.NewModuleAddress(stabletypes.ModuleName)
+
+	depositDenom := k.stableKeeper.GetDepositDenom(ctx)
+
+	balance := k.bankKeeper.GetBalance(ctx, moduleAddr, depositDenom)
+	borrowed := params.TotalValue.Sub(balance.Amount)
+	borrowRatio := sdk.ZeroDec()
+	if params.TotalValue.GT(sdk.ZeroInt()) {
+		borrowRatio = sdk.NewDecFromInt(borrowed).Add(msg.Leverage.Mul(msg.CollateralAmount.ToLegacyDec())).
```

Comment:
I think we need to multiply It with 100 here or use 0.7 in params.MaxLeveragePercent